python/test/test_polars.py

```diff
@@ -94,19 +94,22 @@ def test_expr_ffi():
     assert str(t_ident((lf, pl.col("A")))[1]) == str(pl.col("A"))
 
 
-def test_private_lazyframe_explicit_sum():
+@pytest.mark.parametrize(
+    "measure", [dp.max_divergence(T=float), dp.zero_concentrated_divergence(T=float)]
```

Comment:
This is just as strong as the current state, but it does highlight that there's nothing here that's checking that `max_divergence` and `zero_concentrated_divergence` have any difference in behavior.